### ISSUE TITLE
`commons/colors.tex` not found in package

### ISSUE THREAD

**Issue by TamotsuTokazu** [moderncv/moderncv#194]
**Describe the bug**
I installed moderncv using tlmgr, which says the package repo is [`https://ctan.math.washington.edu/tex-archive/systems/texlive/tlnet`](https://ctan.math.washington.edu/tex-archive/systems/texlive/tlnet). Compiling any document results in an error of `commons/colors.tex` not found.

After some quick investigation, I found the `commons` in the directory for documents (`some-path/doc/latex/moderncv` in my case), rather than where it should be (`some-path/tex/latex/moderncv` in my case). So I assume maybe something's wrong with packaging for distribution?

**Expected Behavior**

**Actual Behavior**

**Minimum Working Example to Reproduce the Problem**

**Specifications**
  - moderncv version and source: v2.4.0
  - LaTeX compiler (pdfLaTeX, XeLaTeX, etc.) and version: 
  - Platform (operating system, LaTeX editor, etc.): WSL2 with Ubuntu 22.04

**Additional context**

**Comment by LLdaniel** [comment]
The package is located at https://mirrors.ctan.org/macros/latex/contrib/moderncv/ and it seems that the folder structure matches the structure of the uploaded `.tar.gz`:
```bash
tar -tvf moderncv-v2.4.0-0-ge8a2096.tar.gz 'moderncv/commons/colors.tex'
-rw-rw-r-- root/root      1516 2024-07-10 07:34 moderncv/commons/colors.tex
```
This in turn matches the directory structure of the source code (besides the `moderncv` prefix created with the tar archive).

May I ask if you are running `tlmgr` in usermode? Also did you install texlive manually or from the Ubuntu repository?
I am trying to understand your current setup in case different versions of texlive or `tlmgr` could be the reason for that problem?

**Comment by marcfrederick** [comment]
After updating the `moderncv` package using `tlmgr`, I'm seeing the same issue:

```
l.25 \input{commons/colors}
                           ^^M
/usr/local/texlive/2024/texmf-dist/tex/latex/moderncv/moderncviconsawesome.sty:25:  ==> Fatal error occurred, no output PDF file produced!
Transcript written on lebenslauf.log.
Latexmk: Missing input file 'commons/colors.tex' (or dependence on it) from following:
  ! LaTeX Error: File `commons/colors.tex' not found.
```

Looking at the tar file from my mirror, the whole commons/ directory seems to be missing: https://mirror.physik.tu-berlin.de/pub/CTAN/systems/texlive/tlnet/archive/moderncv.r71767.tar.xz. This corresponds to the contents of my /usr/local/texlive/2024/texmf-dist/tex/latex/moderncv directory, which is also missing that directory.

### Specifications

- moderncv version and source: 2.4.0
- LaTeX compiler (pdfLaTeX, XeLaTeX, etc.) and version: pdfTeX 3.141592653-2.6-1.40.26 (TeX Live 2024)
- Platform (operating system, LaTeX editor, etc.): macOS 14.5, Latexmk 4.85, VSCode 1.91.1, LaTeX Workshop v10.1.0

**Comment by LLdaniel** [comment]
Thanks to everyone for reporting this issue. 
The problem is now understood: The missing file name prefix for `colors.tex` was responsible for bundling that file into the documentation archive on ctan (for a more detailed explanation see linked PR).

Once this PR is merged I will immediately create a new patch version 2.4.1 for ctan. So bear with me a little more and the patch will be incoming soon...

**Comment by LLdaniel** [comment]
I submitted the new patched version v2.4.1 to CTAN just now. 
So after the distribution to all mirrors it should be available within the next 24h.
